data/TX/incentives.json

```diff
@@ -0,0 +1,351 @@
+[
+  {
+    "id": "TX-1",
+    "payment_methods": [
+      "rebate"
+    ],
+    "items": [
+      "heat_pump_water_heater"
+    ],
+    "program": "tx_AustinEnergy_RebatesAndIncentives",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 800
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$800 rebate for a qualifying ENERGY STAR heat pump water heater."
+    }
+  },
+  {
+    "id": "TX-2",
+    "payment_methods": [
+      "rebate"
+    ],
+    "items": [
+      "other"
+    ],
+    "program": "tx_AustinEnergy_RebatesAndIncentives",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 300
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$300 rebate for qualifying ENERGY STAR variable-speed pool pumps."
+    }
```

Comment:
I think we need to come to a position on whether we need to track incentives at all for items that aren't of interest, like pool pumps. No need to do that here, but we ought to discuss in Slack.

Reply:
Sounds good. I think Sarah added that one, but we can discuss.